scripts/astrokat-targets.py

```diff
@@ -1060,11 +1116,28 @@ def main(creation_time,
     header_info = {'proposal_id': args.prop_id,
                    'pi_name': args.pi,
                    'pi_contact': args.contact}
+    if args.target is not None:
+        target = args.target
+    elif args.body is not None:
+        target = [args.body]
+    elif args.xephem is not None:
+        target = args.xephem
+    else:
+        target = None
```

Comment:
That new fancy Python switch statement would be useful here. Pity, it's only on Py3.10